Services/CO.CDP.Localization/FormsEngineResource.resx

```diff
@@ -150,6 +150,9 @@
   <data name="FinancialInformation_03_Title" xml:space="preserve">
     <value>Were your accounts audited?</value>
   </data>
+  <data name="FinancialInformation_03_Description" xml:space="preserve">
+    <value>&lt;p class="govuk-body"&gt;Auditing requirements are defined in &lt;a href="https://www.legislation.gov.uk/ukpga/2006/46/part/16" target="https://www.legislation.gov.uk/ukpga/2006/46/part/16"&gt;part 16 of the Companies Act 2006 (opens in new tab)&lt;/a&gt; or an overseas equivalent&lt;/p&gt;</value>
```

Comment:
should be `target="_blank"` so that the link opens in a new window

```suggestion
    <value>&lt;p class="govuk-body"&gt;Auditing requirements are defined in &lt;a href="https://www.legislation.gov.uk/ukpga/2006/46/part/16" target="_blank"&gt;part 16 of the Companies Act 2006 (opens in new tab)&lt;/a&gt; or an overseas equivalent&lt;/p&gt;</value>
```